improver/wxcode/weather_symbols.py

```diff
@@ -141,16 +141,18 @@ def check_input_cubes(self, cubes: CubeList) -> Optional[List[str]]:
         required by the decision tree.  Sets self.coord_named_threshold to
         "True" if threshold-type coordinates have the name "threshold" (as
         opposed to the standard name of the diagnostic), for backward
-        compatibility.
+        compatibility. A cubelist containing only cubes of the required
```

Comment:
From a pedantry perspective, sometimes there can be an issue with methods named `check...` doing more than checking. Perhaps to avoid any confusion here, the method could be renamed to something like `prepare`?

Reply:
Changed.